test/async-hooks/test-get-connections.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const assert = require('assert');
+const net = require('net');
+const server = net.createServer();
+
+try {
+    server.getConnections(() => {
+        assert.ok(true, "No error thrown")
+    });
+}
+catch (e) {
+    assert.ok(false, "getConnections threw an error");
+}
```

Comment:
You don’t need the try/catch, if an error is thrown the test is going to fail anyway because the process crashes with an uncaught exception